Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3
+
+# Install b2, curl, perl, jq
+RUN cd /tmp \
+ && wget -O /usr/local/bin/b2 \
+    https://github.com/Backblaze/B2_Command_Line_Tool/releases/latest/download/b2-linux \
```

Comment:
Is there a way to get a specific version of b2? When possible, I think we prefer to lock in the major version number of tools and dependencies.

Reply:
I guess there is.  I was just following the instructions at https://www.backblaze.com/docs/cloud-storage-command-line-tools.